libbeat/scripts/load-dashboards.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+# Usage examples:
+# env KIBANA_INDEX='.kibana_env1' ./load.sh
+# ./load.sh -url http://test.com:9200
+# ./load.sh -url http://test.com:9200 -user admin:secret
+# ./load.sh -url http://test.com:9200 -index .kibana-test
+
+# The default value of the variable. Initialize your own variables here
+ELASTICSEARCH=http://localhost:9200
+CURL=curl
+KIBANA_INDEX=".kibana"
+KIBANA_PATH?=""
```

Comment:
+1 to first move the dashboards as they are. Reason is that they should be in a working state when we cut alpha1.